### ISSUE TITLE
Copying from the source in browse mode

### ISSUE THREAD

**Issue by Gene703122** [nvaccess/nvda#13520]
### Is your feature request related to a problem? Please describe.
When in Browse Mode, NVDA doesn't show paragraph divisions nor does it allow text to be copied and pasted as a sighted person would see it.
### Describe the solution you'd like
There should be an option to copy from the underlying source when copying to the clipboard when in Browse mode.  This should probably be the default behavior.
### Describe alternatives you've considered
I now turn browse mode off, use control a to copy an entire web page, then paste it into a word processor or text editor.  I then edit it.  If I don't do that, I may send someone extended text with no paragraph divisions.  That is a very inefficient work around.
### Additional context
Who knows how many NVDA users send material to sighted people with no idea that it lacks paragraph divisions and that structures such as tables aren't formatted as a sighted person would ordinarily see them.  

I've been told that if sighted people get documents that are not very short with no paragraph divisions that it is very difficult to read and that some people find it so difficult that they don't do so.

**Comment by Gene703122** [comment]
I closed the issue because a sighted person tells me that paragraphs are retained if you paste into a word processor when you copy in browse mode.  The matter needs further clarification.  It may still be that tables and perhaps other things aren't properly formatted.  Once I understand if the problem is serious enough to merit a ticket, I'll create another one, if indicated.

**Comment by Brian1Gaff** [comment]
This is one of those catch 22 things. I rely on a basic copy myself, since 
I'm using the text mostly for making scripts for humans to read, but never 
present them with a table, since they are  hard to make understandable. Yes 
we do need an extra copy, if you really want to do as you say, but how do 
you discover where to start and end if you are only hearing the result? In 
many cases I've used a browser able to show page source and cut and pasted 
that, but that is quite  long winded, I'll grant you.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message ----- 
From: "Gene703122" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Monday, March 21, 2022 4:52 PM
Subject: [nvaccess/nvda] Copying from the source in browse mode (Issue 
#13520)
